Documentation/network/servicemesh/ingress.rst

```diff
@@ -87,6 +87,13 @@ Additionally, cloud-provider specific annotations for the LoadBalancer service
 are supported. Please refer to the `Kubernetes documentation <https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer>`_
 for more details.
 
+.. warning::
+
+    The  ``Ingress`` and ``Gateway API`` features in ``Cilium`` require the layer 7 proxy to be enabled in order to function properly. 
```

Comment:
You probably don't need to mention `Gateway API` on this page, which is only about Ingress.

Reply:
> You probably don't need to mention `Gateway API` on this page, which is only about Ingress.

Alright.